portfolio/src/main/webapp/script.js

```diff
@@ -23,6 +23,28 @@ function addRandomGreeting() {
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
+  const greetingContainer = document.getElementById('greeting');
   greetingContainer.innerText = greeting;
 }
+
+
+/*
+* Handles image replacement on hover
```

Comment:
Since this is a one line comment I'd either change to
`// Handles image replacement on hover `
Or to
`/* Handles image replacement on hover */`